doc/sphinx-guides/source/container/running/demo.rst

```diff
@@ -0,0 +1,132 @@
+Demo or Evaluation
+==================
+
+If you would like to demo or evaluate Dataverse running in containers, you're in the right place. Your feedback is extremely valuable to us! To let us know what you think, please see :ref:`helping-containers`.
+
+.. contents:: |toctitle|
+	:local:
+
+Quickstart
+----------
+
+- Download :download:`compose.yml <../../../../../docker/compose/demo/compose.yml>`
+- Run ``docker compose up`` in the directory where you put ``compose.yml``
+- Visit http://localhost:8080 and try logging in:
+
+  - username: dataverseAdmin
+  - password: admin1
+
+Hardware and Software Requirements
+-----------------------------------
+
+- 8 GB RAM (if not much else is running)
+- Mac, Linux, or Windows (experimental)
+- Docker
+
+Windows support is experimental but we are very interested in supporting Windows better. Please report bugs (see :ref:`helping-containers`).
+
+Tags and Versions
+-----------------
+
+The compose file references a tag called "alpha", which corresponds to the latest released version of Dataverse. This means that if a release of Dataverse comes out while you are demo'ing or evaluating, the version of Dataverse you are using could change. We are aware that there is a desire for tags that correspond to versions to ensure consistency. You are welcome to join `the discussion <https://dataverse.zulipchat.com/#narrow/stream/375812-containers/topic/tagging.20images.20with.20versions/near/366600747>`_ and otherwise get in touch (see :ref:`helping-containers`). For more on tags, see :ref:`supported-image-tags-app`.
+
+Once Dataverse is running, you can check which version you have through the normal methods:
+
+- Check the bottom right in a web browser.
+- Check http://localhost:8080/api/info/version via API.
+
+About the Containers
+--------------------
+
+If you run ``docker ps``, you'll see that multiple containers are spun up in a demo or evaluation. Here are the most important ones:
+
+- dataverse
+- postgres
+- solr
+- smtp
+- bootstrap
+
+Most are self-explanatory, and correspond to components listed under :doc:`/installation/prerequisites` in the (traditional) Installation Guide, but "bootstrap" refers to :doc:`../configbaker-image`.
+
+Additional containers are used in development (see :doc:`../dev-usage`), but for the purposes of a demo or evaluation, fewer moving (sometimes pointy) parts are included.
+
+Security
+--------
+
+Please be aware that for now, the "dev" persona is used to bootstrap Dataverse, which means that admin APIs are wide open (to allow developers to test them; see :ref:`securing-your-installation` for more on API blocking), the "create user" key is set to a default value, etc. You can inspect the dev person `on GitHub <https://github.com/IQSS/dataverse/blob/master/modules/container-configbaker/scripts/bootstrap/dev/init.sh>`_ (look for ``--insecure``).
+
+We plan to ship a "demo" persona but it is not ready yet. See also :ref:`configbaker-personas`.
```

Comment:
I've never created a persona. I hope doesn't take much time to create and document how to use it.